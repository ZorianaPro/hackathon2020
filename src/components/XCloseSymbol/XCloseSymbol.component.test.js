import React from 'react';
import { shallow } from 'enzyme';
import XCloseSymbol from './';

describe('XCloseSymbol', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <XCloseSymbol />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<XCloseSymbol />)).toMatchSnapshot();
  });
});
