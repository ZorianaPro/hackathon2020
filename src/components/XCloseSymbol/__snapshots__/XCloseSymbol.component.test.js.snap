// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`XCloseSymbol renders as expected 1`] = `
<symbol
  className="XCloseSymbol"
  id="XCloseSymbol"
  viewBox="0 0 24 24"
>
  <path
    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
    fill="#727E8B"
    fillRule="nonzero"
    id="XCloseSymbol-Path"
  />
</symbol>
`;
